bios/meekrosoft.md

```diff
@@ -0,0 +1,5 @@
+## Mike Long
+
+Scotstrailian at large.  Minister without portfolio.
+
+Find me on github as @meekrosoft
```

Comment:
The casing of GitHub™ in this file violates the GitHub™ brand.
For more information about GitHub™ brand representation, please visit www.github™.com